users/service.go

```diff
@@ -14,18 +17,41 @@ var (
 	ErrConflict = errors.New("email already taken")
 
 	// ErrMalformedEntity indicates malformed entity specification (e.g.
-	// invalid username or password).
+	// invalid username or password)
```

Comment:
I think that we should have `.` at the end because godoc will generate documentation from these comments. And documentation won't look nice without normal sentences.